repos/system_upgrade/common/actors/checkmountoptions/libraries/checkmountoptions.py

```diff
@@ -0,0 +1,27 @@
+from leapp import reporting
+from leapp.libraries.stdlib import api
+from leapp.models import StorageInfo
+
+
+def check_noexec_on_var(storage_info):
+    """Check for /var being mounted with the noexec option."""
+
+    for mount_entry in storage_info.mount:
+        if mount_entry.mount == '/var' and 'noexec' in mount_entry.options.split(','):
+            reporting.create_report([
+                reporting.Title(
+                    'Mount options used to mount /var are preventing the upgrade'
+                ),
+                reporting.Summary(
+                    'The /var is mounted with the noexec option, preventing the upgrade.'
+                ),
+                reporting.Severity(reporting.Severity.HIGH),
+                reporting.Tags([reporting.Flags.FILESYSTEM]),
```

Comment:
`AttributeError: type object 'Flags' has no attribute 'FILESYSTEM'`

```suggestion
                reporting.Tags([reporting.Tags.FILESYSTEM]),
```

Reply:
Thanks, fixed.